pkg/ytconfig/cri.go

```diff
@@ -0,0 +1,67 @@
+package ytconfig
```

Comment:
Let's add canonized test for exec node spec with new fields. It should be easy since we have a lot of them.